docs/src/Modal.doc.js

```diff
@@ -1,17 +1,26 @@
 // @flow strict
 import type { Node } from 'react';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
 import PageHeader from './components/PageHeader.js';
-import Card from './components/Card.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Modal"
-    description="Modals are light temporary views; general purpose containers allowing for a multitude of design choices."
+    description="Modals display content that requires user interaction on a layer above the page content. Modals block the content underneath, preventing users from interacting with anything else besides the Modal.
+
+    The most common example of Modal is confirming an action the user has taken. Modal also provides some of the padding required for styling out of the box."
+    defaultCode={`
+      <iframe src="https://codesandbox.io/embed/trusting-wu-c8514?fontsize=14&hidenavigation=1&theme=light&view=preview"
+      style={{width: '100%', height:'500px', border:'0', borderRadius: '4px', overflow:'hidden'}}
+      title="Modal Main Example"
+      allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
```

Comment:
Do we need to allow all this? Seems like a possible privacy concern. I don't think any of those should be necessary for our Modal in a sandbox?